plugins/techdocs-backend/src/service/router.ts

```diff
@@ -323,12 +336,8 @@ export function createEventStream(
 }
 
 /**
- * Create a HTTP response. This is used for the legacy non-event-stream implementation of the sync endpoint.
- *
- * @param res - the response to write the event-stream to
- * @returns A tuple of <log, error, finish> callbacks to emit messages. A call to 'error' or 'finish'
- *          will close the event-stream.
- */
+ *  @deprecated use event-stream implementation of the sync endpoint
```

Comment:
Kind of a bummer that this doesn't get captured in the `api-report.md`, eh?  Maybe worth noting in a changeset too.  Something like:

> _Note: in an upcoming release, TechDocs backend's `/sync/:namespace/:kind/:name` endpoint will only respond to `text/event-stream`-based requests. Update any custom code at your organization accordingly._

Reply:
Yeah agree, but why does it not get captured in the api report? I decided to add the deprecated flag anyways. Will add instructions to changeset!